app/controllers/ApplicationController.java

```diff
@@ -23,16 +19,12 @@
 public final class ApplicationController extends SunriseController {
 
     @Inject
-    public ApplicationController(final PlayJavaSphereClient client, final CategoryTree categoryTree, final Configuration configuration) {
-        super(client, categoryTree, configuration);
+    public ApplicationController(final PlayJavaSphereClient client, final CategoryTree categoryTree,
+                                 final Configuration configuration, final TemplateService templateService, final CmsService cmsService) {
+        super(client, categoryTree, configuration, templateService, cmsService);
     }
 
     public F.Promise<Result> index() {
-        return client().execute(ProductProjectionQuery.of(CURRENT)).map(this::showProducts);
-    }
-
-    private Result showProducts(PagedQueryResult<ProductProjection> productPagedQueryResult) {
-        final List<ProductProjection> products = productPagedQueryResult.getResults();
-        return ok(products.toString());
+        return F.Promise.pure(redirect(productcatalog.controllers.routes.ProductCatalogController.pop()));
```

Comment:
For the 303 redirection we need the inverse route (which is converted into a URL) as it is used here, which is independent from the template system used. The TemplateService only knows which template to use for the required page, it knows nothing about URLs.
Notice what I'm doing here is simply redirect the index page to the POP so that Sunrise is initially showing something. This will change once we have the home page designed.